httpref.go

```diff
@@ -64,10 +64,6 @@ func (r References) ByName(code string) References {
 // InRange attempts to find a numeric in a range in the Name field
 func (r References) InRange(code string) References {
 	for _, v := range r {
-		if v.Name == code {
```

Comment:
As explained in the commit message:

> The results were inconsistent since References are not sorted by single-port first and wide port ranges may match before single port ones. This is not a breaking change since the logic of InRange being used in the app checks first for specific ports

This logic is, at least, misleading. Depending on the order of the rules on the `structs` of ports, the return may be one or the other. See for example the rule for the port `19150`. It matches the rule `10000-20000` (line 9484 of registered-ports) instead of the more specific rule of `19150` (line 10970 from registered-ports) because the most general rule appears first.

Since it's a non breaking change and the code is not currently being used, I would remove it to achieve a 100% coverage of the project.